blocks/default-output-block/output-types/default.jsx

```diff
@@ -184,6 +184,7 @@ const SampleOutputType = ({
       <body>
         {comscoreNoScript(comscoreID)}
         {googleTagManagerNoScript(gtmID)}
+        <a className="skip-main" href="#main">Skip to content</a>
```

Comment:
Can we use our language functions here to ensure this text string is localised for all support languages?